apm-agent-plugins/apm-servlet-plugin/src/main/java/co/elastic/apm/agent/servlet/service_name/Servlet2PlusInstrumentation.java

```diff
@@ -16,8 +16,10 @@
  * specific language governing permissions and limitations
  * under the License.
  */
-package co.elastic.apm.agent.servlet;
+package co.elastic.apm.agent.servlet.service_name;
```

Comment:
```suggestion
package co.elastic.apm.agent.servlet.servicename;
```

See also https://stackoverflow.com/questions/3179216/what-is-the-convention-for-word-separator-in-java-package-names